app/src/features/mocks/modals/DeletRecordsModalWrapper/DeleteRecordsModal/index.tsx

```diff
@@ -0,0 +1,58 @@
+import React from "react";
+import { useState } from "react";
+import { RQButton } from "lib/design-system/components";
+import { RQModal } from "lib/design-system/components";
+import { toast } from "utils/Toast";
+import { RQMockMetadataSchema } from "components/features/mocksV2/types";
+import deleteIcon from "../../assets/delete.svg";
+import { useMocksActionContext } from "features/mocks/contexts/actions";
+import { isRecordMock } from "features/mocks/screens/mocksList/components/MocksList/components/MocksTable/utils";
+import "./deleteRecordsModal.scss";
+
+interface Props {
+  visible: boolean;
+  records: RQMockMetadataSchema[];
+  toggleModalVisibility: (visible: boolean) => void;
+  onSuccess?: () => void;
+}
+
+export const DeleteRecordsModal: React.FC<Props> = ({ visible, records, toggleModalVisibility, onSuccess }) => {
+  const [isDeleting, setIsDeleting] = useState(false);
+  const { deleteRecordsAction } = useMocksActionContext() ?? {};
```

Comment:
Instead of pulling this from `MocksActionContext`. Lets just put it here only for consistency and so that DeleteRecordsModal become self independent

Reply:
Similar to `UpdateMocksCollectionModal`. Modal contains UI as well as login